electives/16311.md

```diff
@@ -0,0 +1,59 @@
+# 16-311: Introduction to Robotics
+
+| **Category** | **Difficulty (Out of 5)** |
+| --- | --- |
+| Homework | 2 |
+| Labs | 4 |
+| Exams | 4 |
+
+This course is a part of the Robotics Institute's curriculum for students pursuing a Minor or Additional Major in Robotics.
+
+# Topics Covered
+
+The class covers key topics in robotics such as Controls, Kinematics, Motion Planning, Localisation and Computer Vision.
+
+1. Computer Vision
+2. Neural Networks
+3. Reinforcement Learning
+4. Controls
+5. Motion Planning
+6. Localisation
+7. Forward and Inverse Kinematics
+8. Non Holonomic Path Planning
+9. Motors and Communication Protocols
+10. Batteries and Prototyping
+
+# Class Structure
+
+1. Lectures on the above topics mentioned
+    - Labs are also introduced in Lecture
+2. Guest Lectures
+3. Homeworks 
+    - These are written but can also include programming elements depending on the topic being covered particularily early in the course.
+
+# Homeworks
+
+The homework component typically has a written part designed to cover lecture content/theory and then occasionally an accompanying programming element to apply what you have learned e.g Training a NN in PyTorch when learning about Reinforcement Learning. They aren't designed to be super challenging and are due on Wednesday the day after labs, so it's definitely feasible to Start/Finish them in a day.
+
+# Exams
+
+The class is broken up into a midterm and a final. The exam mainly involves questions from the lecture slides, guest lectures, and homework. There is a bank of past midterms and finals released prior to the midterm/final that can allow you to familiarise yourself with the format and type of questions that you are expected to answer.
+
+# Labs
+
+44% of your grade is tied to the labs so expect this part of the class to occupy the lion's share of the time you dedicate to it. Most Labs are completed in self-selected groups sized 2-3, but some are individual/pair based. I would strongly reccomend forming a group of 3. You are encouraged to have students from different majors ECE/CS/Mech-E to share your expertise in different areas, but it's not required. Labs typically span a week, with a demo period on Tuesday. They can vary quite significantly in difficulty, but leveraging the help from TAs and starting early makes them manageable. 
+
+Since demos are on Tuesday, I personally would suggest starting the labs at the latest on Saturday. The labs typically have 2-3 components tied to them. Designing a physical robot out of the lego BrickPi Kit, Programming and sometimes tuning constants through experimentaton/trial and error.
```

Comment:
```suggestion
Since demos are on Tuesday, I personally would suggest starting the labs at the latest on Saturday. The labs typically have 2-3 components tied to them. Designing a physical robot out of the lego BrickPi Kit, Programming and sometimes tuning constants through experimentation/trial and error.
```